cmd/posix.go

```diff
@@ -285,6 +310,81 @@ func (s *posix) checkDiskFound() (err error) {
 	return err
 }
 
+// stat information in parallel.
+func statInRoutine(dirPath string, entries []string, doneCh <-chan struct{}) <-chan os.FileInfo {
+	fInfoCh := make(chan os.FileInfo, 1)
+	go func() {
+		defer close(fInfoCh)
+		for _, entry := range entries {
+			fi, _ := os.Stat(pathJoin(dirPath, entry))
```

Comment:
Why are we ignoring error here?

Reply:
Nothing to do with errors... at this point.